prep-javascript-variables/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullname = 'Marlon Adams';
+    console.log(fullname);
+    var numberOfPets = '3';
+    console.log(numberOfPets);
+    var likesvegetables = 'false';
+    Boolean(likesvegetables);
+    console.log(likesvegetables);
+    var nothing = 'null';
```

Comment:
Hi Marlon, be sure to leave out the quotes so that the JavaScript engine doesn't interpret your numbers, booleans, and null as string values.